payload-schemas/schemas/repository.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "repository",
+
+  "type": "object",
+  "required": ["action", "repository", "sender"],
+  "properties": {
+    "action": {
+      "type": "string",
+      "enum": [
+        "created",
+        "deleted",
+        "archived",
+        "unarchived",
+        "edited",
+        "renamed",
+        "transferred",
+        "publicized",
+        "privatized"
+      ]
+    },
+    "changes": {
+      "type": "object",
+      "properties": {},
+      "additionalProperties": true
+    },
```

I'd say let's be more strict/explicit. We can expand the schema as we add more payload examples, see the discussion at https://github.com/octokit/webhooks/pull/235

```suggestion
    "changes": {
      "type": "object", 
      "properties": {
        "description": {
          "type": "object", 
          "properties": {
            "from": {
              "type": "string"
            }
          },
          "additionalProperties": false
        }
      },
      "additionalProperties": false
    }
```

---

The discussion moved forward and I happened to overlook this, sorry. Anyways, it looks like that with the effort of self-updating schemas underway it'd be OK to leave it as-is. Please let me know what you think.